src/styles/styles.css

```diff
@@ -70,3 +70,144 @@ html {
 .social-medias__icon {
   width: 100%;
 }
+
+.higtlight-card-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(30.3125rem, 1fr));
+  grid-gap: 1.9rem;
+  list-style: none;
+  padding: 0;
+}
+
+.higtlight-card,
+.higtlight-card-second {
+  position: relative;
+}
+
+.higtlight-card::before,
+.higtlight-card-second::before {
+  border-radius: 10px;
+  content: "";
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  z-index: 20;
+}
+.higtlight-card::before {
+  background-color: rgba(215, 215, 98, 0.6);
+}
+
+.higtlight-card-second::before {
+  background-color: rgba(185, 82, 59, 0.6);
+}
+
+.higtlight-card__image {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  display: block;
+  top: 0;
+  left: 0;
+  z-index: 10;
+  object-fit: cover;
+  border-radius: 10px;
+}
+
+.higtlight-info__card__image {
+  width: 3.125rem;
```

Comment:
Se essa for a imagem do `author` coloque um `height: 3.125rem` para garantir que a imagem fique sempre quadrada.